### ISSUE TITLE
Accessible names for tables

### ISSUE THREAD

**Issue by aviannamiller** [omeka/omeka-s#2395]
Tables on Omeka S are often, if not always, described by the information provided in a page's `<h1>`. However, the tables themselves lack the accessible names necessary to communicate their intended titles directly to AT. Each table should implement some form of an `aria-label`, as modeled quite nicely in https://github.com/omeka-s-modules/ArchivesspaceConnector/issues/1#issuecomment-3648115619.
